src/sagemaker_mxnet_container/serving.py

```diff
@@ -72,7 +69,7 @@ def default_model_fn(model_dir, preferred_batch_size=1):
 
     sym, args, aux = mx.model.load_checkpoint(os.path.join(model_dir, DEFAULT_MODEL_NAME), 0)
 
-    ctx = _context()
+    ctx = mx.eia() if os.environ.get(INFERENCE_ACCELERATOR_PRESENT_ENV) == 'true' else _context()
```

Comment:
why move this out of the helper function?

Reply:
because the helper function is used below where `mx.eia()` can't be called. Why keep the helper function? in case we ever do make good on the todo that's in there.